http/api_handler.go

```diff
@@ -125,10 +125,6 @@ func NewAPIHandler(b *APIBackend, opts ...APIHandlerOptFn) *APIHandler {
 
 	h.Mount("/api/v2", serveLinksHandler(b.HTTPErrorHandler))
 
-	authorizationBackend := NewAuthorizationBackend(b.Logger.With(zap.String("handler", "authorization")), b)
```

Comment:
How does this ladder up to `idpe`? Removing this here will cause the handlers to be missing if someone updates the `influxdb` dependency. I think we'll need to make this tolerant of that scenario.

Reply:
this is how I added it in idpe: https://github.com/influxdata/idpe/pull/7054/files#diff-f047f7de48d9a985ade16057d9d48739R109-R124

Reply:
Okay, so the intention is that you will merge this then *immediately* merge the other? I would be careful with this assumption in some cases, because there could be some delay in your ability to merge that's out of your control. Would it be possible to merge the `idpe` changes first?